packages/plugin-essentials/sources/commands/set/version/sources.ts

```diff
@@ -106,7 +106,7 @@ export default class SetVersionCommand extends BaseCommand {
       const bundlePath = ppath.resolve(target, `packages/yarnpkg-cli/bundles/yarn.js` as PortablePath);
       const bundleBuffer = await xfs.readFilePromise(bundlePath);
 
-      await setVersion(project, `yarn-sources`, bundleBuffer, {
+      await setVersion(project, `sources`, bundleBuffer, {
```

Comment:
`yarn-` already added in https://github.com/yarnpkg/berry/blob/694e03756b7017a5718d5cbe5ae49bd6e6d8c27e/packages/plugin-essentials/sources/commands/set/version.ts#L203